File: C#/백준/Bronze/25629. 홀짝 수열/홀짝 수열.cs
using System;

namespace q{
    class w{
        static void Main(String[] args){
            int a = int.Parse(Console.ReadLine());
            int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            int cnt = 0;
            for (int i=0;i<a;i++){
                int n = aa[i]%2;
                if (n==1){
                    cnt++;
                }
                else{
                    cnt--;
                }
            }
            if (a%2==cnt){
                Console.WriteLine("1");
            }
            else{
                Console.WriteLine("0");
            }
        }
    }
}

File: Python/백준/Bronze/25629. 홀짝 수열/홀짝 수열.py
a = int(input())
cnt = 0
aa = list(map(int,input().split()))
for i in aa:
    if i%2==0:
        cnt+=1
    else:
        cnt-=1
if a%2==0:
    if cnt==0:
        print(1)
    else:
        print(0)
else:
    if cnt==-1:
        print(1)
    else:
        print(0)